offchain/authority-claimer/src/signer/signer.rs

```diff
@@ -56,6 +56,7 @@ impl ConditionalSigner {
                 mnemonic,
                 account_index,
             } => {
+                tracing::info!("signing method: mnemonic");
```

Comment:
I think this is unnecessary since we added the auth_kind config var. See https://github.com/cartesi/rollups-node/blob/main/internal/node/config/generate/Config.toml#L145

Reply:
I'll drop the commit